web/src/user_groups_settings_ui.js

```diff
@@ -19,6 +19,11 @@ import * as user_groups from "./user_groups";
 
 let group_list_widget;
 
+// Ideally this should be included in page params.
+// Like we have page_params.max_stream_name_length` and
+// `page_params.max_stream_description_length` for streams.
+export const max_user_group_name_length = 100;
```

Comment:
@laurynmm can I add doing this API tweak to your TODO list? It's not urgent but also probably quick, so probably best do wait until we have a moment with fewer big API changes about to merge.